GLPT_Cocoa.inc

```diff
@@ -1,89 +1,390 @@
+
+procedure NSShow (obj: id);
+begin
+  NSLog(obj.description, NSSTR('@'));
+end;
+
+procedure Fatal (msg: string);
```

Comment:
Replace function by glptError. For now I would create a new constant eg GLPT_ERROR_FATAL. 

Although not part of this commit, I think glptError needs to be redesigned and simplified anyway. Only a few error levels are needed eg FATAL, WARNING, INFO. Not sure if this is covering everything though. Depending on a global constant or compiler define the errors could be redirected to the system log as well.